docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -534,26 +540,32 @@ USER 1001
 CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
 ----
 
+NOTE: This multi-stage Docker builds copies the Maven Wrapper from the host machine.
```

Comment:
`Docker` -> `Docker build`

Reply:
So, it was wrong, but with a different mistake :-) "build" instead of "builds".
The exact wording is "multi-stage Docker build" (https://docs.docker.com/develop/develop-images/multistage-build/).

Reply:
Ah, Okay 👌